DotnetCLIVersion.txt

```diff
@@ -1 +1 @@
-2.1.200
\ No newline at end of file
+2.1.301
```

Comment:
Let's revert this change for now.  We needed it when we were using `netcoreapp2.1`, but now that we are using `netcoreapp3.0`, this change isn't necessary.

Reply:
Edited thanks